.gitignore

```diff
@@ -12,6 +12,8 @@
 # See the License for the specific language governing permissions and
 # limitations under the License.
 
+go.work
+go.work.sum
```

Comment:
@ChristopherFry decided to exclude `go.work` and `go.work.sum` to keep the changes outside the `rollouts` directory to minimal.
I can share these from my workspace.